pkg/security/podsecuritypolicy/provider.go

```diff
@@ -310,6 +316,10 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		}
 	}
 
+	if !s.psp.Spec.AllowPrivilegeEscalation && sc.AllowPrivilegeEscalation != nil && *sc.AllowPrivilegeEscalation {
```

Comment:
we don't have a default we set we just have a behavior though when it is nil (proposal: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/no-new-privs.md#pod-security-policy-changes), if I understand your comment correctly, so it is allowed to be nil here I think